ocaml-cohttp-lwt/bin/main.ml

```diff
@@ -0,0 +1,26 @@
+open Ocaml_cohttp_lwt
+
+let () =
+  let port = 8080 in
+  let scenarios =
+    [
+      Scenarios.scenario_1;
+      Scenarios.scenario_2;
+      Scenarios.scenario_3;
+      Scenarios.scenario_4;
+      Scenarios.scenario_5;
+      Scenarios.scenario_6;
+      Scenarios.scenario_7;
+      Scenarios.scenario_8;
+      Scenarios.scenario_9;
```

Comment:
```suggestion
      Scenarios.scenario_9;
      Scenarios.scenario_10;
```

Reply:
Looks like maybe this isn't needed. But not sure why 10 is outside the list.

Reply:
The scenario_10 wasn't returning a string, but I've now changed that to make the list more consistent.